src/ape/api/explorers.py

```diff
@@ -48,8 +49,20 @@ def get_contract_type(self, address: AddressType) -> Optional[ContractType]:
         Get the contract type for a given address if it has been published in an explorer.
 
         Args:
-            address (str): The contract address.
+            address (:class:`~ape.types.AddressType`): The contract address.
 
         Returns:
-            :class:`~ape.contracts.ContractType` if published, else ``None``.
+            Optional[:class:`~ape.contracts.ContractType`]: If not published, returns ``None``.
+        """
+
+    @abstractmethod
+    def get_account_transactions(self, address: AddressType) -> Iterator[ReceiptAPI]:
```

Comment:
Making this an `Iterator` because who the heck knows how many transactions one has.
In the `ape-etherscan`, we have to paginate through requests (the API token is going to be necessary, so that is next after https://github.com/ApeWorX/ape-etherscan/pull/12).

Reply:
Like this, can handle pagination via iterator to smooth it out